### PR TITLE
add dl query tutorial

### LINKED ISSUE #114: DL query tutorial
Putting this ticket out to see if anyone has a good tutorial/how-to/reference for DL queries (rather not reinvent the wheel and all) 
Tagging people who might have one/might know where to find one (please help me tag others if you think they might too).
@dosumis @cmungall @matentzn @nicolevasilevsky @sabrinatoro

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #114
Direct copy from https://ontology101tutorial.readthedocs.io/en/latest/EXERCISE_BasicDL_Queries.html

### CODE REVIEW

File: docs/tutorial/basic-dl-query.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# DL query
+
+This tutorial is based off https://ontology101tutorial.readthedocs.io/en/latest/DL_QueryTab.html
+Created by: Melissa Haendel, Chris Mungall, David Osumi-Sutherland, Matt Yoder, Carlo Torniai, and Simon Jupp
+
+## DL query tab
+
+The DL query tab shown below provides an interface for querying and searching an ontology. The ontology must be classified by a reasoner before it can be queried in the DL query tab.
+
+For this tutorial, we will be using cc.owl which can be found [here](https://github.com/OHSUBD2K/BDK14-Ontologies-101/tree/master/BDK14_exercises/basic-dl-query).
```
Comment (matentzn) [inline docs/tutorial/basic-dl-query.md:10]: Please move any such file dependencies in obook itself, by creating a `dl-query` folder in the `tutorial` folder and place any ontologies or other resources in there. OBOOK should be as self-contained as possible.

File: docs/tutorial/basic-dl-query.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Go to the “basic-dl-query” folder and open “cc.owl”. Run the reasoner. Navigate to the DL Query tab.
```
Comment (matentzn) [inline docs/tutorial/basic-dl-query.md:12]: For folders and filenames in general (not necessary, just as a convention) I kinda prefer `tick-syntax` over "quotes".. Up to you though.

File: docs/tutorial/basic-dl-query.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Type “organelle” into the box, and make sure “subclasses” and “direct subclasses” are ticked.
+
+![](../images/tutorials/dl-query/Figure58.png)
+
+You can type any valid OWL class expression into the DL query tab. For example, to find all classes whose members are part_of a membrane, type “part_of some membrane” and click ‘execute’. Note the linking underscore for this relation in this ontology. Some ontologies do not use underscores for relations, whereby you’d need single quotes (i.e. ‘part of’).
+
+![](../images/tutorials/dl-query/Figure59.png)
+
+The OWL keyword “and” can be used to make a class expression that is the intersection of two class expressions. For example, to find the classes in the red area below, we want to find subclasses of the intersection of the class ‘organelle’ and the class ‘endoplasmic reticulum part’
```
Comment (matentzn) [inline docs/tutorial/basic-dl-query.md:24]: Your quotes look suspicious, like from a different character encoding. Shouldnt they look more like " (more 'straight')?

File: mkdocs.yml [hunk 4]
```diff
@@ -47,6 +47,7 @@ nav:
               - ODK - Managing imports: tutorial/managing-dynamic-imports-odk.md
               - ODK - Managing ontology releases: tutorial/managing-ontology-releases-odk.md
               - ODK - Migrating to ODK: tutorial/migrating-ontology-to-odk.md
+              - DL queries: tutorial/basic-dl-query.md
```
Comment (matentzn) [inline mkdocs.yml:50]: Maybe better in Protege section rather than ODK?

**Review by matentzn — CHANGES_REQUESTED** [review]

File: docs/tutorial/basic-dl-query.md (hunk 1 above)
Comment (shawntanzk) [reply docs/tutorial/basic-dl-query.md:10]: pulled over all the files in https://github.com/OBOAcademy/obook/pull/116, once that is merged I'll redirect this

File: docs/tutorial/basic-dl-query.md (hunk 3 above)
Comment (shawntanzk) [reply docs/tutorial/basic-dl-query.md:24]: will go through and change it all

File: docs/tutorial/basic-dl-query.md (hunk 2 above)
Comment (shawntanzk) [reply docs/tutorial/basic-dl-query.md:12]: honestly didnt think too much about it, just copied and pasted as is lol

File: mkdocs.yml (hunk 4 above)
Comment (shawntanzk) [reply mkdocs.yml:50]: whoops, my bad, totally agree

**Comment by shawntanzk** [comment]
has dependences in #116

**Comment by matentzn** [comment]
@shawntanzk what is your timeline of finishing this? We are thinking of using this for the next OBO academy call.

**Comment by shawntanzk** [comment]
not a rush at all, just discussed docs with @dosumis and we are going to push out the curator pathway first and will leave these and others that im working on as in progress.

**Comment by matentzn** [comment]
The earlier the better :D

**Comment by shawntanzk** [comment]
ohoh it's ready for review now

**Review by matentzn — APPROVED** [review]
No content review, just technically this looks good.

Since this tutorial is basically a copy, I think we are good to merge.

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
I approved too. :)

**Pull request merged**